### PR TITLE
Add Twitch - Users - Get Users function

### PR DESCRIPTION
**Pull request by JulianS-Uni**
I added a function for the Get Users function described here
https://dev.twitch.tv/docs/v5/reference/users/#get-users

### CODE REVIEW

File: twitch/api/users.py [hunk 1]
```diff
@@ -12,6 +12,10 @@ class Users(TwitchAPI):
     def get(self):
         response = self._request_get('user')
         return User.construct_from(response)
+      
+    def get_by_name(self, user_id):
+        response = self._request_get('users?login={}'.format(user_id))
```
Comment (tsifrer) [inline twitch/api/users.py]: `_request_get` accepts a url an params, where params attribute is a dictionary containing get parameters.
So this like should be something like
```
response = self._request_get('users', params={'login': user_name})
```

File: twitch/api/users.py [hunk 2]
```diff
@@ -12,6 +12,10 @@ class Users(TwitchAPI):
...
+        return User.construct_from(response)
```
Comment (tsifrer) [inline twitch/api/users.py]: The documentation says that it returns multiple users (or one user inside a list), so you need to extract each user and convert it to User object
```
 return [User.construct_from(x) for x in response['users']]
```

File: twitch/api/users.py (hunk 1 above)
Comment (JulianS-Uni) [reply twitch/api/users.py]: I just copied the syntax from get_by_id() more or less. When your code creates a URL like this
`https://api.twitch.tv/kraken/users?login=<user IDs>`
this should be fine. However, how would you create a URL with multiple user names like this
`https://api.twitch.tv/kraken/users?login=dallas,dallasnchains`

File: twitch/api/users.py (hunk 2 above)
Comment (JulianS-Uni) [reply twitch/api/users.py]: Yes you are right. That looks good.

File: twitch/api/users.py (hunk 1 above)
Comment (tsifrer) [reply twitch/api/users.py]: Ah its that kind of URL param.
I'd suggest then doing something like
```
def get_by_name(self, user_ids):
    if not isinstance(user_ids, list):
        user_ids = list(user_ids)
    params = {
        'login': ','.join(user_ids)
    }
    response = self._request_get('users', params=params)
```

Comment (TechplexEngineer) [reply twitch/api/users.py]: In python 3.7.3 `isinstance("user_name", list)` is true.

To make your suggestion work:
```python
def get_by_name(self, user_ids):
        if isinstance(user_ids, str):
            user_ids = [user_ids]
        params = {
            'login': ','.join(user_ids)
        }
        response = self._request_get('users', params=params)
        return User.construct_from(response)
```

**Comment by hellno** [comment]
Hi! I'm super interested in the functionality in this PR.
What is the status here? Can I help in some way?
Thanks :)

**Pull request closed without merging**